client/src/css/App.css

```diff
@@ -2,6 +2,10 @@
   text-align: center;
 }
 
+ul, ol {
```

Comment:
keep in mind that these are global styles. eventually it may make more sense to separate out style logic into their own files pertaining their respective components.

furthermore, a lot of this styling can also be done with the bootstrap library we implemented a while back. feel free to take a look through their docs to see how the classes their could better serve our UI in order to avoid custom styling

Reply:
Yep. This styling was implemented for visual purposes in testing.